spring-cloud-skipper-server-core/src/test/resources/repositories/sources/test/python/python-printer-1.0.1/templates/vanakkam.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: skipper.spring.io/v1
 kind: CloudFoundryApplication
 spec:
-  resource: "{{{spec.resource}}}"
+  resource: https://github.com/ilayaperumalg/sandbox/raw/master/python/1.0.1/hello.py-{{spec.version}}.zip
```

Comment:
I suppose to use version in a real life would not work because you have hardcoded base dirs, `1.0.0` and `1.0.1`.